docs/coordinates/performance.inc.rst

```diff
@@ -56,3 +56,157 @@ properties::
 
 ..
   EXAMPLE END
+
+
+Improving Performance for Arrays of ``obstime``
+-----------------------------------------------
+
+The most expensive operations when transforming between observer-dependent coordinate
+frames (e.g. ``AltAz``) and sky-fixed frames (e.g. ``ICRS``) are the calculation
+of the orientation and position of Earth.
+
+If |skycoord| instances with a large ``obstime`` array are transformed,
+these calculations can be sped up by factors up to 100, whilst still keeping micro-arcsecond precision,
+by utilizing interpolation instead of calculating Earth orientation parameters for each individual point.
+
+This can be achieved through the ``erfa_astrom`` state and the ``ErfaAstromInterpolator``
+class like this
+
+..
+  EXAMPLE START
+  Improving performance for obstime arrays
+
+To use interpolation for the astrometric values in coordinate transformation, use::
+
+   >>> from astropy.coordinates import SkyCoord, EarthLocation, AltAz
+   >>> from astropy.coordinates.erfa_astrom import erfa_astrom, ErfaAstromInterpolator
+   >>> from astropy.time import Time
+   >>> from time import perf_counter
+   >>> import numpy as np
+   >>> import astropy.units as u
+
+
+   >>> # array with 10000 obstimes
+   >>> obstime = Time('2020-01-01T20:00') + np.linspace(0, 6, 10000) * u.hour
+   >>> location = location = EarthLocation(lon=-17.89 * u.deg, lat=28.76 * u.deg, height=2200 * u.m)
+   >>> frame = AltAz(obstime=obstime, location=location)  # doctest:+REMOTE_DATA
+   >>> crab = SkyCoord.from_name('Crab')  # doctest:+REMOTE_DATA
+
+   >>> # transform with default transformation and print duration
+   >>> t0 = perf_counter()
+   >>> crab_altaz = crab.transform_to(frame)  # doctest:+REMOTE_DATA +IGNORE_WARNINGS
+   >>> print(f'Transformation took {perf_counter() - t0:.2f} s')  # doctest:+ELLIPSIS
```

Comment:
How about `+IGNORE_OUTPUT` here and then give the actual number? The point here is more to show how much it helps than to get the doctest to pass!

Reply:
I thought the actual output would still be reported, but of course it can't be.